R/composeVar.R

```diff
@@ -62,8 +75,9 @@ composeVar.GADSdat <- function(GADSdat, sourceVars, primarySourceVar, newVar) {
   ## put into GADS, add meta
   dat_out <- data.frame(GADSdat$dat, comp_var, stringsAsFactors = FALSE)
   names(dat_out)[ncol(dat_out)] <- newVar
-  GADSdat_out <- updateMeta(GADSdat, newDat = dat_out)
-  GADSdat_out2 <- reuseMeta(GADSdat_out, varName = newVar, other_GADSdat = GADSdat_out, other_varName = primarySourceVar, addValueLabels = TRUE)
+  suppressMessages(GADSdat_out <- updateMeta(GADSdat, newDat = dat_out, checkVarNames = FALSE))

```

Comment:
Indeed. `checkVarNames()` is already explicitly called earlier within the function. `updateMeta()` is rather verbose and provides additional messages that should be suppressed here.

In the future, `updateMeta()` might be revised so this could be simplified.